packages/frontend/src/components/UnderlyingData/UnderlyingDataProvider.tsx

```diff
@@ -101,7 +104,7 @@ export const UnderlyingDataProvider: FC<Props> = ({
 }) => {
     const [config, setConfig] = useState<UnderlyingDataConfig>();
     const [isModalOpen, setModalOpen] = useState<boolean>(false);
-
+    const { data: explore } = useExplore(tableName);
```

Comment:
Does it work with on non explorable contexts like dashboards ?

Reply:
Yes, `useExplore` is just a request to the BE. You might be confused with `useExplorer` which is a deprecated hook to the explorer provider.

Reply:
You're totally right